pkg/trace/remoteconfighandler/agent_failover.go

```diff
@@ -0,0 +1,30 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2025-present Datadog, Inc.
+
+package remoteconfighandler
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+// multiRegionFailoverConfig is a deserialized Multi-Region Failover configuration file
+type multiRegionFailoverConfig struct {
+	FailoverMetrics *bool `json:"failover_metrics"`
+	FailoverLogs    *bool `json:"failover_logs"`
```

Comment:
Do we care about those in the context of APM?

Reply:
We do not. I mostly kept them to keep full parity with the core agent implementation. Having 2 different implementations of the remote config client is already confusing. I will comment them out and add a comment there  to avoid deserializing those values 👍

Reply:
Why have two separate implementations?. Can the trace agent not use the RC client used in Core?